APolitcalApp/src/views/ViewProfilePage.vue

```diff
@@ -13,8 +36,23 @@
       <ToolbarProfilePersonal></ToolbarProfilePersonal>
     </template>
     <template #ContentSlot>
-      <h1>profile</h1>
-      <AuthAccount @sign-out="authStore.logout()" v-if="authStore.session" :session="authStore.session" />
+      <div class="flex flex-col gap-5">
+        <AuthAccount @sign-out="authStore.logout()" v-if="authStore.session" :session="authStore.session" />
+        <div class="flex w-full justify-between">
+          <div>
+            <p>Your Petitions</p>
+            <div class="flex flex-col gap-2">
+              <CardPetitionTitle v-for="petition in petitions" :key="petition.id" :petition-id="petition.id" :petition-title="petition.title" />
+            </div>
+          </div>
+          <div>
+            <p>Your Solutions</p>
+            <div class="flex flex-col gap-2">
+              <CardSolutionsOtherSuggestions v-for="post in solutions" :linkedPetition="post.id" :suggestionText="post.description" :uservotes="post.uservotes.length ?? 0" />
```

Comment:
I am getting the console error TypeError: Cannot read properties of null (reading 'length') and none of the solutions render. Not sure why the ?? isn't working.